main.tf

```diff
@@ -51,7 +57,7 @@ module "seed_project" {
   random_project_id           = true
   disable_services_on_destroy = false
   folder_id                   = var.folder_id
-  org_id                      = google_organization_iam_member.tmp_project_creator.org_id
```

Comment:
This is intentionally setup this way, so that project creator rights are added, before we try to create the project - we may need a local, that changes the behavior based on if its a folder or not

Reply:
@rjerrems makes sense, I have modified it such that if it will wait for either `google_organization_iam_member` or `google_organization_iam_member` to resolve before proceeding with the project creation.